src/components/Appointments/AppointmentDetails/AppointmentDetails.tsx

```diff
@@ -1,30 +1,120 @@
-import { Route } from "react-router";
+import { useEffect, useState } from "react";
+import { NavLink, useParams } from "react-router-dom";
+import { useDispatch } from "react-redux";
+import { setError } from "../../../redux/reducers/AppointmentSlice";
 
-import { Space, Card } from "antd";
-import { useCustomSelector } from "../../../Redux/Store";
+import { getCurrentAppointmentApi, updateAppointmentApi } from "../../../api";
+import { convertPhoneNumber } from "../../../utils/convertPhoneNumber";
+import StatusesSelect from "../../StatusesSelect";
+import ModalDelete from "../../Modals/ModalDelete";
+import Loading from "../../Loading";
 
-import styles from "./Appointments.module.scss";
+import { Space, Card, Button } from "antd";
+import styles from "./AppointmentDetails.module.scss";
 
 export default function AppointmentDetails() {
```

Comment:
is it is page (you put this component into Route component) then it is better when is finishes with Page:
AppointmentDetailsPage